### PR TITLE
fix: no-shortcut-reference-link

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: locale/en/blog/release/v0.10.45.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Commits:
 * [[`8df4b0914c`](https://github.com/nodejs/node/commit/8df4b0914c)] - **deps**: separate sha256/sha512-x86_64.pl for openssl (Shigeki Ohtsu) [joyent/node#25654](https://github.com/joyent/node/pull/25654)
 * [[`11eefefb17`](https://github.com/nodejs/node/commit/11eefefb17)] - **deps**: copy all openssl header files to include dir (Shigeki Ohtsu) [#6553](https://github.com/nodejs/node/pull/6553)
 * [[`61ccc27b54`](https://github.com/nodejs/node/commit/61ccc27b54)] - **deps**: upgrade openssl sources to 1.0.1t (Shigeki Ohtsu) [#6553](https://github.com/nodejs/node/pull/6553)
-* [[`aa02438274`](https://github.com/nodejs/node/commit/aa02438274)] - **openssl**: fix keypress requirement in apps on win32 (Shigeki Ohtsu) [joyent/node#25654](https://github.com/joyent/node/
+* [[`aa02438274`](https://github.com/nodejs/node/commit/aa02438274)] - **openssl**: fix keypress requirement in apps on win32 (Shigeki Ohtsu) [joyent/node#25654](https://github.com/joyent/node/pull/6553)
```
Comment (XhmikosR) [inline locale/en/blog/release/v0.10.45.md]: I don't think this is right. It should be https://github.com/joyent/node/pull/25654

**Review by XhmikosR — CHANGES_REQUESTED** [review]

File: locale/en/blog/release/v0.10.45.md (hunk 1 above)
Comment (nschonni) [reply locale/en/blog/release/v0.10.45.md]: I'm not sure if the trees were merged when the repos were rejoined. Are you suggesting pointed to the IO.js repo for these?

Comment (XhmikosR) [reply locale/en/blog/release/v0.10.45.md]: Sorry, I meant https://github.com/joyent/node/pull/25654

Comment (XhmikosR) [reply locale/en/blog/release/v0.10.45.md]: Damn, GitHub changes the link automatically :P

Comment (XhmikosR) [reply locale/en/blog/release/v0.10.45.md]: This `https://github.com/joyent/node/pull/25654`

**Comment by XhmikosR** [comment]
@nschonni can you fix tests and take care of my comment above?

**Review by MylesBorins — COMMENTED** [review]
The changelogs are generated from the upstream changelog in the repo itself. I'm not 100% that we should be changing them for a linting rule

**Comment by Trott** [comment]
> The changelogs are generated from the upstream changelog in the repo itself. I'm not 100% that we should be changing them for a linting rule

Since we're importing them from elsewhere, it may be best to simply skip linting on them entirely and enforce any lint rules we want upstream (or just don't worry about the linting for the changelogs).

**Comment by nschonni** [comment]
Although a bunch of the changelog related fixes have been applied upstream, i've just disabled the same rules plus this new one at the file level for the changlog blogs

File: locale/en/blog/release/v0.10.42.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ slug: node-v0-10-42
 layout: blog-post.hbs
 author: James M Snell
 ---
+<!--lint disable prohibited-strings-->
```
Comment (XhmikosR) [inline locale/en/blog/release/v0.10.42.md:10]: Can you confirm such comments are not in the built HTML? Not a blocker, but we should be aware about it.

File: locale/zh-cn/docs/guides/simple-profiling.md [hunk 3]
```diff
@@ -139,7 +139,7 @@ node --prof-process isolate-0xnnnnnnnnnnnn-v8.log > processed.txt
    3165    8.4%    8.6%  _malloc_zone_malloc
 ```
 
-我们看到，前 3 个条目占了程序占用的 CPU 时间的 72.1%。从这个输出中，我们立即看到至少 51.8% 的 CPU 时间被称为 PBKDF2 的函数占用。它与用户密码中的哈希生成相对应。然而，较低的两个条目的因素是如何进入我们的应用程序（或者我们为了例子而假装如此）不会立即明显得看出来。为了更好地理解这些函数之间的关系，接下来我们将查看[自下而上（重）配置文件]部分，该节提供有关每个函数的主要调用方的信息。检查此部分，我们会发现：
+我们看到，前 3 个条目占了程序占用的 CPU 时间的 72.1%。从这个输出中，我们立即看到至少 51.8% 的 CPU 时间被称为 PBKDF2 的函数占用。它与用户密码中的哈希生成相对应。然而，较低的两个条目的因素是如何进入我们的应用程序（或者我们为了例子而假装如此）不会立即明显得看出来。为了更好地理解这些函数之间的关系，接下来我们将查看\[自下而上（重）配置文件\]部分，该节提供有关每个函数的主要调用方的信息。检查此部分，我们会发现：
```
Comment (XhmikosR) [inline locale/zh-cn/docs/guides/simple-profiling.md:142]: This is Chinese to me so I can't judge the change 😛

**Comment by XhmikosR** [comment]
LGTM now, it needs a rebase and I left a couple of comments. I can approve after those are addressed 🙂

File: locale/en/blog/release/v0.10.42.md (hunk 2 above)
Comment (nschonni) [reply locale/en/blog/release/v0.10.42.md:10]: Yes, it looks like it ends up the built HTML

**Comment by nschonni** [comment]
Rebased. Confirmed that the `remark-lint-no-literal-urls` will need additional work in a separate PR to re-enable

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**